config/routes.rb

```diff
@@ -1,5 +1,10 @@
 Rails.application.routes.draw do
 
+  # Adding these routes connects those requests to the appropriate actions of the errors controller.
+  # Using match ... :via => :all allows the error pages to be displayed for any type of request (GET, POST, PUT, etc.)
+  match "/404", :to => "errors#not_found", :via => :all
+  match "/500", :to => "errors#internal_server_error", :via => :all
```

Comment:
same here the style for hashes is without the hashrocket when we can `to: "errors#not_found", via: :all` for example